frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
+
+    .commands__cli {
+        height: 50vh;
+        overflow: auto;
+        padding: 2vh 1vw;
+        @extend %gradientBackground, %blackGradientBackground;
+
+        .command {
+            color:color(blue);
+        }
+        .success {
+            color:color(green);
+        }
+        .error {
+            color: color(red);
+        }
+        p {
+            margin: 0;
+        }
+    }
+}
+.form {
+    position: relative;
+    color: color(white);
+    .form__input {
```

Comment:
No need to create a class here. You can just style directly after `input[type="text"]` as we know its just one.